CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 - Added sensitivity calculation support for `pybamm.Simulation` and `pybamm.Experiment` ([#4415](https://github.com/pybamm-team/PyBaMM/pull/4415))
 - Added OpenMP parallelization to IDAKLU solver for lists of input parameters ([#4449](https://github.com/pybamm-team/PyBaMM/pull/4449))
-- Added phase-dependent particle options to LAM #4369
+- Added phase-dependent particle options to LAM
+  ([#4369](https://github.com/pybamm-team/PyBaMM/pull/4369))
```

Comment:
```suggestion
- Added phase-dependent particle options to LAM ([#4369](https://github.com/pybamm-team/PyBaMM/pull/4369))
```

Reply:
Both of those render why does this matter?

Reply:
I think it would split up into a new line on https://pybamm.org/changelog/

Reply:
It is already a bulleted list in that page, if that does not work then there is something wrong with the library used for it

Reply:
Now that I notice, that page is outdated and does not show any results since 24.1 was last released

Reply:
In other words, the Goldmark renderer for Hugo has an option to wrap text, will check if it's enabled